### PR TITLE
Documentation fixes and tweaks

### PR DESCRIPTION
**Pull request by ericpre**
### Progress of the PR
- [x] Fix icons in navigation bar,
- [x] fix formatting changelog,
- [x] fix logo in `README.md`,
- [n/a] update user guide (if appropriate),
- [n/a] add an changelog entry in the `upcoming_changes` folder (see [`upcoming_changes/README.rst`](https://github.com/hyperspy/rosettasciio/blob/main/upcoming_changes/README.rst)),
- [x] Check formatting changelog entry in the `readthedocs` doc build of this PR (link in github checks)
- [n/a] add tests,
- [x] ready for review.

### CODE REVIEW

File: upcoming_changes/76.doc.rst [hunk 1]
```diff
@@ -1 +1 @@
-Consolidate docstrings and documentation for all plugins (see also `#47 <https://github.com/hyperspy/rosettasciio/issues/47`_, `#59 <https://github.com/hyperspy/rosettasciio/issues/59`_, `#64 <https://github.com/hyperspy/rosettasciio/issues/64`_, `#72 <https://github.com/hyperspy/rosettasciio/issues/72`_)
+Consolidate docstrings and documentation for all plugins (see also `#47 <https://github.com/hyperspy/rosettasciio/pull/47>`_, `#59 <https://github.com/hyperspy/rosettasciio/issues/59>`_, `#64 <https://github.com/hyperspy/rosettasciio/issues/64>`_, `#72 <https://github.com/hyperspy/rosettasciio/issues/72>`_)
```
Comment (jlaehne) [inline upcoming_changes/76.doc.rst:1]:
```suggestion
Consolidate docstrings and documentation for all plugins (see also `#47 <https://github.com/hyperspy/rosettasciio/pull/47>`_, `#59 <https://github.com/hyperspy/rosettasciio/pull/59>`_, `#64 <https://github.com/hyperspy/rosettasciio/pull/64>`_, `#72 <https://github.com/hyperspy/rosettasciio/pull/72>`_)
```

**Comment by jlaehne** [comment]
`docs/readthedocs.org:rosettasciio` Link to documentation build from this PR is missing in the checks:
https://rosettasciio--88.org.readthedocs.build/en/88/

@ericpre, maybe this PR is a good occasion to add that again?

**Comment by ericpre** [comment]
> `docs/readthedocs.org:rosettasciio` Link to documentation build from this PR is missing in the checks: https://rosettasciio--88.org.readthedocs.build/en/88/
> 
> @ericpre, maybe this PR is a good occasion to add that again?

I have tried to reset the integration to github but it doesn't fix it... I don't know...

**Review by jlaehne — APPROVED** [review]

**Comment by jlaehne** [comment]
Weird, because it is still working for the other repos.

**Pull request merged**